### PR TITLE
React Capstone Project, Capstone #3

### PR DESCRIPTION
**Pull request by jmonto55**
This PR contains all the steps made to build the [React Capstone Project](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md), including:

[LOOM VIDEO](https://www.loom.com/share/28734c69306943c091d3aca6fe22105f)

- Selection of [Financial modeling prep API](https://financialmodelingprep.com/developer/docs/) to get stock data.
- When the page loads, the webapp shows the list of markets that could be filtered out by company name or symbol.
- Display some numeric values that come from the API.
- When the user clicks (or taps) on a market or stock, the application navigates to the details page.
- In the details page, the webapp retrieves data from the API to show detailed data in the category.
- When the user clicks on the "Back" button (<), the user navigates to the home page.
- Creation of unit tests for pure functions in the Redux code.
- Creation of integration tests using the React Testing Library.
- The project is a single-page application (SPA) built with React and Redux.
- The data retrieved from the API is stored in the Redux store.
- You should filter the data that you retrieve from the API using a Filter stateless component.
- Every page (the main page and the pages for each item) has a unique route within the SPA.

### CODE REVIEW

**Comment by binhussen** [comment]
@jmonto55 would add the video presenation, to continue the review

File: README.md [hunk 1]
```diff
@@ -137,14 +136,14 @@ npm test
...
-- GitHub: [@anderCM](https://github.com/anderCM)
-
 👤 **Jose Montoya**
 
 - GitHub: [@jmonto55](https://github.com/jmonto55)
 
+👤 **Nelson Sakwa**
+
+- Behance: [@jmonto55](https://www.behance.net/sakwadesignstudio)
+
```
Comment (binhussen) [inline README.md]: - [Optional] Great Job for adding the author to the design :+1: , however, this `author` section was prepared only for those who contributed to this repo, so add it to the "Acknowledgements" section [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) :+1:

File: README.md [hunk 2]
```diff
@@ -68,18 +67,18 @@
...
+- **The application is fully responsive.**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Live Demo](https://space-travelers-hub-27nq.onrender.com/)
+- [Live Demo](https://stock-viewer.onrender.com/)
```
Comment (binhussen) [inline README.md:81]: - Kindly, you forget to add the video presentation :+1: , please add it either in your pr description or in the readme as it is part of the [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md)
Screenshot from 2023-03-17 00-08-11

**Review by binhussen — CHANGES_REQUESTED** [review]
Hi @jmonto55  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- Used External api ✔️
- Used rreact-redux✔️ 
- Added integration test✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (jmonto55) [reply README.md]: Thank you, @binhussen

File: README.md (hunk 2 above)
Comment (jmonto55) [reply README.md:81]: My bad, I just updated the PR with the video, thank you 👌

**Review by DuaneDave — APPROVED** [review]
Hi @jmonto55 👋 , 

In my opinion, your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Comment by jmonto55** [comment]
Thanks, @DuaneDave

**Pull request merged**